kernel/common/get_slice.rs

```diff
@@ -1,8 +1,6 @@
-use core::slice;
-use core::str;
 use collections::range::RangeArgument;
 use core::ops::Range;
-use core::cmp::{max, min};
+use core::cmp::min;
```

Comment:
That is weird then. You shouldn't have to make those changes in `get_slice.rs` as least, since I already took care of them a long time ago :stuck_out_tongue_closed_eyes: